layers/eight_mile/pytorch/layers.py

```diff
@@ -2839,7 +2905,10 @@ def forward(self, qkvm: Tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Te
         key = self.w_K(key).view(batchsz, -1, self.h, self.d_k).transpose(1, 2)
         value = self.w_V(value).view(batchsz, -1, self.h, self.d_k).transpose(1, 2)
 
-        rpr_key, rpr_value = self.make_rpr(seq_len, query.device)
+        if self.ra_masking:
+            rpr_key, rpr_value = self.get_rpr(query.device)
```

Comment:
the naming here is confusing.  should be more descriptive, when we are using a limited horizon mask, we have `get_rpr` and when we use Shaw style, we have `make_rpr`.  How about something `make_windowed_rpr()`?